v0/destinations/marketo/util.js

```diff
@@ -91,14 +99,13 @@ const marketoResponseHandler = (destResponse, sourceMessage, stage) => {
   }
   // non 2xx failure
   throw new ErrorBuilder()
-    .setStatus(status)
+    .setStatus(400)
```

Comment:
Why we are making it to 400 as we can send whatever response we are getting from the destination?

Reply:
We are getting `200` status from destination. Any successful should've already been returned from **[here](https://github.com/rudderlabs/rudder-transformer/pull/1432/files#diff-6d523239215a17c74a801bf8be57c380ebdc658bd1a763c98a2faf9e76021e8fR92-R93)**

So it doesn't make sense to send the status as is. Let me know if it makes sense